### ISSUE TITLE
Can not remap office and emoji keys on microsoft keyboards

### ISSUE THREAD

**Issue by Cyrille-de-Brebisson** [randyrants/sharpkeys#538]
Hello,

I have a microsoft ergo keyboard, which has 2 stupid keys.
An "office key", which gets you directly to the microsoft office webpage! What a value!
And an emoji keyboard key. But beeing a professional, I have no use for such childish thing!

So, I tried to remap them with sharpkeys, but when I click on add key/type key and then click on either of these keys, it displays EO_205B and then tells me it does not know said keys...

Anything you can do to add support for these keys in the next version?

Thanks,
Cyrille

**Comment by DreymaR** [comment]
No. These are three-byte codes and as clearly stated in the program documentation it is impossible to remap such keys using the registry method (which SharpKeys is based on). Please re-read the documentation to understand this better.
